pkg/controllers/deprovisioning/multimachineconsolidation.go

```diff
@@ -18,19 +18,24 @@ import (
 	"context"
 	"fmt"
 	"math"
+	"time"
 
 	"k8s.io/apimachinery/pkg/util/sets"
 	"k8s.io/utils/clock"
 	"knative.dev/pkg/logging"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
+	"github.com/samber/lo"
+
 	"github.com/aws/karpenter-core/pkg/cloudprovider"
 	"github.com/aws/karpenter-core/pkg/controllers/provisioning"
 	"github.com/aws/karpenter-core/pkg/controllers/provisioning/scheduling"
 	"github.com/aws/karpenter-core/pkg/controllers/state"
 	"github.com/aws/karpenter-core/pkg/events"
 )
 
+const MultiMachineConsolidationTimeoutDuration = 1 * time.Minute
```

Comment:
nit: Have you considerd having all the timeout in one file? Maybe in the consolidation controller?

Reply:
I'm gonna opt to keep it here, don't think it should really need to be propagated elsewhere.